test/e2e/common/container_probe.go

```diff
@@ -413,10 +416,14 @@ var _ = framework.KubeDescribe("Probing container", func() {
 		startedTime, err := GetContainerStartedTime(p, "busybox")
 		framework.ExpectNoError(err)
 
+		startedIn := readyTime.Sub(startedTime)
 		framework.Logf("Container started at %v, pod became ready at %v", startedTime, readyTime)
-		if readyTime.Sub(startedTime) < 40*time.Second {
+		if startedIn < sleepBeforeStarted*time.Second {
```

Comment:
copying comment from the closed PR. This is a change - original test was comparing with the smaller time (`40`) than the `sleep` duration (`45`). Presumably accounting for some delays. Just want to make sure this will not introduce flakiness